apps/nextjs/src/lib/polyfills/performance-now.ts

```
@@ -0,0 +1,12 @@
+// src/lib/polyfills/performance-now.ts
+declare let Date: DateConstructor;
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)